system/Router/RouteCollection.php

```diff
@@ -1490,4 +1491,21 @@ public function shouldUseSupportedLocalesOnly(): bool
     {
         return $this->useSupportedLocalesOnly;
     }
+
+    /**
```

Comment:
Unless you have a compelling reason for sticking this at the bottom I think it would make more organizational sense to be up above `discoverRoutes()`.

Reply:
Done.